### PR TITLE
feat(docs):  add section on managing repo categories

### PR DESCRIPTION
**Pull request by vwheeler63**
I couldn't find documentation on this anywhere, and it is really important for my long list of repositories I manage.  So I added this section for posterity.

Thank you for making it easy to contribute!

### CODE REVIEW

File: source/dashboard.rst [hunk 1]
```diff
@@ -62,3 +62,48 @@ Clone Github repository
 -----------------------
 
 See :doc:`github`.
+
+Organizing your repositories with categories
+--------------------------------------------
+
+Over time, the list of repositories you deal with can grow long.  As it does, it can become difficult to
```
Comment (gerhardol) [inline source/dashboard.rst:69]: Please use one space after period

File: source/dashboard.rst [hunk 2]
```diff
@@ -62,3 +62,48 @@ Clone Github repository
...
+
+Create new category
+^^^^^^^^^^^^^^^^^^^
+
+While the Dashboard is being displayed (after application start-up), you create a new repository category
+by doing the following:
+
+- right click the repository you want to group into a category,
+- select "Categories",
+- select "Add new..." from the resulting fly-out menu,
```
Comment (RussKie) [inline source/dashboard.rst:82]:
```suggestion
- select "Add new..." from the context menu,
```

**Review by RussKie — APPROVED** [review]

File: source/dashboard.rst [hunk 3]
```diff
@@ -62,3 +62,48 @@ Clone Github repository
...
+- type the name of the new category,
+- click the [OK] button.
+
+The originally-right-clicked repository will now additionally appear in your new category (repository group),
+where you can quickly find it again later no matter where it appears in the "Recent repositories" list.
+
+Rename a category
+^^^^^^^^^^^^^^^^^
+If you later decide you want a category to have a different name:
```
Comment (gerhardol) [inline source/dashboard.rst:91]: insert newline after header

**Review by gerhardol — COMMENTED** [review]
just a few nits

File: source/dashboard.rst (hunk 2 above)
Comment (vwheeler63) [reply source/dashboard.rst:82]: Hi, @RussKie !

Actually, only the "Categories" item is on the context menu.  Another fly-out menu is required to be accessed to be able to select the "Add new..." item.

File: source/dashboard.rst (hunk 1 above)
Comment (vwheeler63) [reply source/dashboard.rst]: Okay!  Done throughout the section in subsequent commit.

File: source/dashboard.rst (hunk 3 above)
Comment (vwheeler63) [reply source/dashboard.rst:92]: Okay!  Done for both sub-headings in subsequent commit.

File: source/dashboard.rst (hunk 2 above)
Comment (gerhardol) [reply source/dashboard.rst:82]: slightly more in style - used before in the doc
```suggestion
- select "Add new..." from the resulting submenu,
```

Thanks for the contribution. The doc need a lot more of care...

**Comment by gerhardol** [comment]
Thanks for the contribution!

**Comment by vwheeler63** [comment]
> Thanks for the contribution!

My pleasure, @gerhardol !  😄

**Comment by vwheeler63** [comment]
@gerhardol P.S.  Will it show up in [the documentation](https://git-extensions-documentation.readthedocs.io/en/release-5.0/dashboard.html) soon?

**Comment by gerhardol** [comment]
> @gerhardol P.S.  Will it show up in [the documentation](https://git-extensions-documentation.readthedocs.io/en/release-5.0/dashboard.html) soon?

It should be in the 5.1 branch

**Comment by vwheeler63** [comment]
> It should be in the 5.1 and main branches

It is indeed!  Awesome!

**Pull request merged**